h/search/query.py

```diff
@@ -22,21 +22,20 @@ def wildcard_uri_is_valid(wildcard_uri):
     """
     Return True if uri contains wildcards in appropriate places, return False otherwise.
 
-    *'s are not permitted in the scheme or netloc. _'s are not permitted in the scheme.
+    *'s and _'s are not permitted in the scheme or netloc aka:
+        scheme://netloc/path;parameters?query#fragment.
```

Comment:
Might be worth adding a note here about the reason for disallowing wildcards in the scheme/netloc; to limit the performance impact from Elasticsearch requests which match an extremely large percentage of terms in a field's term dictionary.